zimbra/src/main/java/fr/openent/zimbra/controllers/ZimbraController.java

```diff
@@ -1261,19 +1261,20 @@ public void zimbraEventBusHandler(Message<JsonObject> message) {
                         Boolean hasExpertRight = WorkflowActionUtils.hasRight(user, WorkflowActions.EXPERT_ACCESS_RIGHT.toString());
                         if (Boolean.TRUE.equals(hasExpertRight)) {
                             calendarServiceImpl.getICal(user)
-                                    //TODO format ical
-                                    .onSuccess(result -> {
-                                        //TODO send formatted ical back
+                                    .onSuccess(ical -> {
+                                        ical = ical.replaceAll("\\\r\\\n", "\n");
+                                        ical = ical.replaceAll(";TZID\\=.*\\/.*\\\"", "");
```

Comment:
remove `;TZID=\"Europe/Brussels\"` from ical (timezone will be treated in [MC-279](https://jira.support-ent.fr/browse/MC-279) )